index.html

```diff
@@ -0,0 +1,89 @@
+
+    <link rel="stylesheet" href="styles.css">
+    <title>YouTube Replica</title>
+</head>
+<body>
+
+    <header class="header">
+        <div class="logo left">
+            <i id="menu" class="material-icons">menu</i>
+            <img src="images/icons8-youtube-47.png">
+        </div>
+
+        <div class="search centre">
+            <form action="">
+                <input type="text" placeholder="Search"/>
+                <button><i class="material-icons">search</i></button>
+            </form>
+            <i class="material-icons">mic</i>
+        </div>
+
+        <div class="icons right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons">account_circle</i>
+        </div>
+
+    </header>
+    <main>
+        <div class="side-bar">
+        <div class="nav">
+            <a class="nav-link active">
+                <i class="material-icons">home</i>
+                <span>Home</span>
+            </a>
+
+            <a class="nav-link">
+                <i class="material-icons">local_fire_department</i>
+                <span>Trending</span>
+            </a>
+
+            <a class="nav-link">
+                <i class="material-icons">subscriptions</i>
+                <span>Subscription</span>
+            </a>
+
+        </div>
+        </div>
+        <div class="content">
+            
+            
+            <div class="title">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/YGVnERMwG8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                <h3>
+                    Mexican Cartels Are Worse Than You Think <br>By Task & Purpose
+
+                </h3>
+            </div>
+            
+            <div class="title">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/Z-pT0XDYvDM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+            
+                <h3>
+                    Inside Google's Massive Headquarters <br> Tech Vision
+                </h3>
+            </div>
+
+            <!-- <div class="title">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/TcqAu8VyjZA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                <h3>
+                    Uzbekistan - The Silk Road by train | DW Documentary <br> DW Documentary
+                </h3>
+            </div> -->
```

Comment:
Please do not leave unused code as a comment in your work. This makes your code unclean.:+1: